.github/workflows/earthly-build.yaml

```
@@ -2,13 +2,13 @@ name: Earthly build
 on:
   workflow_dispatch:
   push:
-    branches: ['main']
-    tags: ['*']
+    branches: ["main"]
+    tags: ["*"]
   pull_request:
   schedule:
-    - cron: '0 11 * * *'
+    - cron: "0 11 * * *"
 jobs:
-  build-testing-results:
+  build-test:
```

Very nit, but maybe for consistency just set the `VCS_REF` like you do in the other jobs:

```suggestion
  build-test:
    env:
       VCS_REF: ${{ github.sha }}
```

And then use the env var in the earthly command below

---

I missed this out.

---

Thanks! Can you also update line 25 to use the var?

```
earthly --ci --output +build-test --VCS_REF="$VCS_REF"
```